From d378c5eb18b7c208307ad5aaafacf91c4f0d73bb Mon Sep 17 00:00:00 2001
From: Alexander Holler <holler@ahsoftware.de>
Date: Thu, 4 Feb 2010 05:50:06 +0100
Subject: [PATCH 04/14] DockStar: Change prompt and ident string.

It just looks better if the greeting reflects the actual used hardware. ;)
---
 include/configs/sheevaplug.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/sheevaplug.h b/include/configs/sheevaplug.h
index cccd142..ecb794c 100644
--- a/include/configs/sheevaplug.h
+++ b/include/configs/sheevaplug.h
@@ -28,7 +28,7 @@
 /*
  * Version number information
  */
-#define CONFIG_IDENT_STRING	"\nMarvell-Sheevaplug"
+#define CONFIG_IDENT_STRING	"\nPlugApps Pogoplug"
 
 /*
  * High Level Configuration Options (easy to change)
@@ -81,7 +81,7 @@
 #define CONFIG_INITRD_TAG	1	/* enable INITRD tag */
 #define CONFIG_SETUP_MEMORY_TAGS 1	/* enable memory tag */
 
-#define	CONFIG_SYS_PROMPT	"Marvell>> "	/* Command Prompt */
+#define	CONFIG_SYS_PROMPT	"PlugApps>> "	/* Command Prompt */
 #define	CONFIG_SYS_CBSIZE	1024	/* Console I/O Buff Size */
 #define	CONFIG_SYS_PBSIZE	(CONFIG_SYS_CBSIZE \
 		+sizeof(CONFIG_SYS_PROMPT) + 16)	/* Print Buff */
-- 
1.6.4.4

